public/index.html

```diff
@@ -29,15 +29,5 @@
   <body>
```

Comment:
You deleted the favicon and logo and they are used here. Use github icon as favicon and use github logo for logo. 

Explore if you don't know about favicons.

Reply:
icon is just a graphical symbol, whereas a logo can have text